packages/editor-ui/src/components/CredentialEdit/CredentialEdit.vue

```diff
@@ -212,21 +212,14 @@ export default mixins(showMessage, nodeHelpers).extend({
 	},
 	async mounted() {
 		this.requiredCredentials =
-			this.uiStore.modals[CREDENTIAL_EDIT_MODAL_KEY].requiredCredentials === true;
+			(this.uiStore.modals[CREDENTIAL_EDIT_MODAL_KEY] as NewCredentialsModal)
```

Comment:
should avoid type conversions (`as` here).. suggestion above to set the types should fix this..

Reply:
Fixed by better typing as said.